.github/workflows/release.yml

```diff
@@ -37,11 +37,7 @@ jobs:
         env:
           GOOGLEAPIS_DIR: "./googleapis"
         run: |
-          go install \
-              github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-grpc-gateway \
-              github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-openapiv2 \
-              google.golang.org/protobuf/cmd/protoc-gen-go \
-              google.golang.org/grpc/cmd/protoc-gen-go-grpc
+          make protoc-plugins
```

Comment:
Why not make this a requirement for `protoc` in the makefile?

Reply:
`make protoc` should be called for every proto file change, calling `go install ...` every time would take much more time

Reply:
Then use make files properly and only build what is needed. For example:
```
# Protoc is used to generate code from a protobuf file
PROTOC_VERSION=3.11.4
PROTOC_ZIP=protoc-$(PROTOC_VERSION)-linux-x86_64.zip
ifeq ($(GOHOSTOS)_$(GOHOSTARCH),darwin_amd64)
    PROTOC_ZIP=protoc-$(PROTOC_VERSION)-osx-x86_64.zip
endif
PROTOC_RELEASES_URI=https://github.com/protocolbuffers/protobuf/releases/download
PROTOC_DOWNLOAD=$(PROTOC_RELEASES_URI)/v$(PROTOC_VERSION)/$(PROTOC_ZIP)

tools/protoc = $(BIN_DIR)/protoc
$(tools/protoc):
	mkdir -p "$(@D)"
	curl --fail --output "$@.zip" -L "$(PROTOC_DOWNLOAD)"
	go run tools/unzip.go "$@.zip" "$@" "bin/protoc"
	rm -f "$@.zip"
```